app/assets/stylesheets/statement.scss

```diff
@@ -0,0 +1,54 @@
+.statement {  
```

Comment:
I prefer if you re-use the path to `app/assets/stylesheets/pages/statements.scss`. That way you don't need to add as part of the pre-compilation process.